src/app/components/search/SearchResults.js

```diff
@@ -230,10 +230,11 @@ function SearchResultsComponent({
         oldQuery.range.updated_at ||
         oldQuery.range.media_published_at ||
         oldQuery.range.report_published_at || {};
-      if ((!datesObj.start_time && !datesObj.end_time) && (datesObj.condition !== 'less_than')) {
+      const relativeCondition = ['less_than', 'more_than'].indexOf(datesObj.condition);
```

Comment:
I suggest changing this comparison to `['less_than', 'more_than'].includes(datesObj.condition)` and adjusting the logic accordingly, for better readability

Reply:
Done